src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -30937,11 +31496,11 @@ public void MaybeNull_ReturnValue_02_WithNotNull()
 @"using System.Diagnostics.CodeAnalysis;
 class Program
 {
-    [return: MaybeNull, NotNull] static T F1<T>(T t) => t;
+    [return: MaybeNull, NotNull] static T F1<T>(T t) => t; //
```

Comment:
>// [](start = 59, length = 2)

Was typing `// -1` a bridge too far? :)

Reply:
I could not bring myself to do it :-P

---
In reply to: [368177057](https://github.com/dotnet/roslyn/pull/40789#discussion_r368177057) [](ancestors = 368177057)